### ISSUE TITLE
computed output size would be negative      [[Node: pool_3 = AvgPool[T=DT_FLOAT, data_format="NHWC", ksize=[1, 8, 8, 1], padding="VALID", strides=[1, 1, 1, 1], _device="/job:localhost/replica:0/task:0/cpu:0"](mixed_10/join)]]  

### ISSUE THREAD

**Issue by AndyLin0128** [tensorflow/tensorflow#12795]
python train model and run bazel-bin/tensorflow/python/tools/optimize_for_inference  --input=/Users/andylin/Desktop/tess/retrained_graph.pb  --output=/Users/andylin/Desktop/tess/retrained_graph_optimized.pb  --input_names=Mul --output_names=final_result.
error:
computed output size would be negative      [[Node: pool_3 = AvgPool[T=DT_FLOAT, data_format="NHWC", ksize=[1, 8, 8, 1], padding="VALID", strides=[1, 1, 1, 1], _device="/job:localhost/replica:0/task:0/cpu:0"](mixed_10/join)]]

**Comment by SragAR** [comment]
Facing the same error.
How did you solve this?

**Comment by xiaohk** [comment]
@SragAR @AndyLin0128 I got the same error as well. Can you share your solution?

**Comment by sonu107** [comment]
@AndyLin0128 @SragAR  @xiaohk  Facing the same error.  Any suggestions

**Comment by VinACE** [comment]
image_shape=(224, 224, 3)  check the input size of your image...  also if you are pretrained weights, make sure you have the shapes mentioned correctly
